bin/Main.purs

```diff
@@ -61,6 +80,11 @@ parser =
         do
           GenerateOperators <$> pursGlobs
             <* Arg.flagHelp
+    , Arg.command [ "generate-rc" ]
+        "Writes a .tidyrc file to the current working directory based\non the command line options given."
```

Comment:
How awful would it be to track what CLI options they provided vs. what got defaulted, so as to only push the flags they provided into the file? For example:

```purs
purs-tidy generate-rc --arrow-first
```

produces a file containing only

```json
{ "typeArrowPlacement": "arrow-first" }
```

...rather than an entire config?